repo.nix

```diff
@@ -1,61 +1,118 @@
+      url = "https://oss.sonatype.org/content/repositories/snapshots/io/iohk/scalanet_2.13/0.5.1-SNAPSHOT/scalanet_2.13-0.5.1-SNAPSHOT-javadoc.jar";
+      sha256 = "71797F6D52041EF84F9A85A93F7DC5AB7AB869FB080CD7D26C61CE517BBCAFDC";
     };
-    "nix-Sonatype OSS Snapshots/io/iohk/scalanet_2.12/0.4.4-SNAPSHOT/scalanet_2.12-0.4.4-SNAPSHOT-sources.jar" = {
-      url = "https://oss.sonatype.org/content/repositories/snapshots/io/iohk/scalanet_2.12/0.4.4-SNAPSHOT/scalanet_2.12-0.4.4-SNAPSHOT-sources.jar";
-      sha256 = "DF9427DE3A85351921504CD755D187ECD043190EFB022B71696D271E10318F1E";
+    "nix-Sonatype OSS Snapshots/io/iohk/scalanet_2.13/0.5.1-SNAPSHOT/scalanet_2.13-0.5.1-SNAPSHOT-sources.jar" = {
+      url = "https://oss.sonatype.org/content/repositories/snapshots/io/iohk/scalanet_2.13/0.5.1-SNAPSHOT/scalanet_2.13-0.5.1-SNAPSHOT-sources.jar";
+      sha256 = "2663CE2354A62D2CADEA5306BEC4C7B56CF05083AF177F2EBD72086D6BF85C05";
     };
-    "nix-Sonatype OSS Snapshots/io/iohk/scalanet_2.12/0.4.4-SNAPSHOT/scalanet_2.12-0.4.4-SNAPSHOT.jar" = {
-      url = "https://oss.sonatype.org/content/repositories/snapshots/io/iohk/scalanet_2.12/0.4.4-SNAPSHOT/scalanet_2.12-0.4.4-SNAPSHOT.jar";
-      sha256 = "101AB6DD48B8454D8843944296D77CD8D4FB4505146F8D65BD99C605637CD984";
+    "nix-Sonatype OSS Snapshots/io/iohk/scalanet_2.13/0.5.1-SNAPSHOT/scalanet_2.13-0.5.1-SNAPSHOT.jar" = {
+      url = "https://oss.sonatype.org/content/repositories/snapshots/io/iohk/scalanet_2.13/0.5.1-SNAPSHOT/scalanet_2.13-0.5.1-SNAPSHOT.jar";
+      sha256 = "548C395BED28F56BFDFC3A52200F113B8C2DE1D9B640A5155D951A4E9A1DF47C";
     };
-    "nix-Sonatype OSS Snapshots/io/iohk/scalanet_2.12/0.4.4-SNAPSHOT/scalanet_2.12-0.4.4-SNAPSHOT.pom" = {
-      url = "https://oss.sonatype.org/content/repositories/snapshots/io/iohk/scalanet_2.12/0.4.4-SNAPSHOT/scalanet_2.12-0.4.4-SNAPSHOT.pom";
-      sha256 = "230C01429D812D40EA57DAFDAD2A8E72A1F1168601D14BFB2573738F2D4B4201";
+    "nix-Sonatype OSS Snapshots/io/iohk/scalanet_2.13/0.5.1-SNAPSHOT/scalanet_2.13-0.5.1-SNAPSHOT.pom" = {
+      url = "https://oss.sonatype.org/content/repositories/snapshots/io/iohk/scalanet_2.13/0.5.1-SNAPSHOT/scalanet_2.13-0.5.1-SNAPSHOT.pom";
+      sha256 = "C370A336DC985116131FAA3DDFC0D37A25E13C7388C7AB60D46E8DF1AB1F6535";
     };
-    "nix-public/ch/megard/akka-http-cors_2.12/1.1.0/akka-http-cors_2.12-1.1.0-javadoc.jar" = {
-      url = "https://repo1.maven.org/maven2/ch/megard/akka-http-cors_2.12/1.1.0/akka-http-cors_2.12-1.1.0-javadoc.jar";
-      sha256 = "44CEF93345FA7A12EB1830F7D7AA9881EAB09D87B5E1E4D10608CECDCC53258B";
+    "nix-local-preloaded/file:/var/lib/buildkite-agent/.sbtix/preloaded/com/google/protobuf/protobuf-java/3.7.0/protobuf-java-3.7.0-sources.jar" = {
+      url = "file:/var/lib/buildkite-agent/.sbtix/preloaded/com/google/protobuf/protobuf-java/3.7.0/protobuf-java-3.7.0-sources.jar";
+      sha256 = "116ADC64F2E67A9F2961C410B1FBE029447C4B39C62F6AE93797989E38C85B83";
```

Comment:
protobufs paths were changes from relative:
```
"nix-public/com/google/protobuf/protobuf-java/3.4.0/protobuf-java-3.4.0-javadoc.jar" = {	
      url = "https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/3.4.0/protobuf-java-3.4.0-javadoc.jar";	
      sha256 = "AD810CD0D6786E656C8BF3C0EA9851BCD3FBA2CCBC1E803326B4012BE0943D19";	
    };
```
to absolute:
```
url = "file:/var/lib/buildkite-agent/.sbtix/preloaded/com/google/protobuf/protobuf-java/3.7.0/protobuf-java-3.7.0-sources.jar";
```
This needs to be investigated.

Reply:
Good catch! I will arrange a call about that also

Reply:
sbt was updated in the past, and it was downgraded because it generated local paths: https://github.com/input-output-hk/mantis/pull/708/files

Maybe restoring sbt version would fix this.

Reply:
I guess it is not an sbt but an sbtix failure?  WDYT?